### DISCUSSION TITLE
Configuring "See code" links

### DISCUSSION THREAD

**Discussion by wilcoxmd** [oclif/oclif#1383] [Q&A]
Hi! When I generate documentation with `oclif readme`, the generated file contains `See code` links that always point to my dist folder. My `oclif.commands` setting is 'dist/commands', so that makes sense to me. 

When I look at the oclif repository though, I see those "See code" links are pointing to source code ([example](https://github.com/oclif/oclif/blob/7e2a62401053061f189404b34b3670013c8f72b5/docs/generate.md#L63)), even thought the [`oclif.commands`](https://github.com/oclif/oclif/blob/7e2a62401053061f189404b34b3670013c8f72b5/package.json#L86) setting looks like it points to the build output. 

How can I configure my documentation generation so that my source code / typescript files are linked instead of the build files (which won't be present on github)?

**Comment by mdonnalley** [comment]
This is the block of code responsible for figuring out the code path: https://github.com/oclif/oclif/blob/4.8.8/src/commands/readme.ts#L308-L332

The `outDirRegex` on line 319 and line 328 are what attempt to replace the configured `outDir` (in your case, `dist`) with `src`.

A couple of things that might be going wrong for you:
- you don't have `typescript` a dependency or devDependency
- your source directory is something other that `src`

Feel free to share your repository - I'm happy to help you figure it out
